### DISCUSSION TITLE
Minimed mobile and Xdrip +

### DISCUSSION THREAD

**Discussion by Thomasr205** [NightscoutFoundation/xDrip#2609] [Q&A] [answered]
Bonjour.

J'ai besoin d'aide pour utiliser l'application minimed mobile avec Xdrip +.

J'ai la dernière version de xdrip, en mode companion app. J'ai l'application minimed mobile qui collecte bien les données de la pompe medtronic 780g.

Mais, xdrip+ n'arrive pas à collecter les données de minimed mobile.

Je mets en pièce jointe des copie d'écran du besoin.

D'avance, merci de votre aide. 

![IMG_20230107_133328](https://user-images.githubusercontent.com/122183583/211173160-cf6da4f4-6713-482a-8868-cf6a38eb067f.jpg)
![IMG_20230107_214112](https://user-images.githubusercontent.com/122183583/211173163-56401575-417d-48d7-b256-7b9613ac0ac0.jpg)
![IMG_20230107_214117](https://user-images.githubusercontent.com/122183583/211173164-c23fc2e4-5b94-4ede-abdb-501f0ebb4769.jpg)
![IMG_20230107_214218](https://user-images.githubusercontent.com/122183583/211173165-4dbd9db8-5df4-4032-86c3-daa0f80f195b.jpg)

**Comment by Navid200** [comment, marked as answer]
This is what Google Translate shows for the post above by @Thomasr205 :


I have the latest version of xdrip, in companion app mode. I have the minimald mobile app which collects data from the medtronic 780g pump fine.

But, xdrip+ fails to collect data from minimal mobile.

I am attaching screenshots of the requirement.

Thank you in advance for your help.

**Reply by psonnera** [reply]
Confirm issue with french app Minimed Mobile 1.3.2.
Is there a tag to add in debug to see which notifications are not seen as legit by xDrip+?
Still, @Thomasr205 can you confirm this behaviour is also there when you do have data in Minimed (your picture shows no data).

**Reply by psonnera** [reply]
![image](https://user-images.githubusercontent.com/33809361/211921346-3365ad5c-751e-4807-a172-5fffb6350a42.png)
![image](https://user-images.githubusercontent.com/33809361/211921505-4b5aa937-5d80-438d-abbb-612d6cb1f6e3.png)

**Reply by Navid200** [reply]
@psonnera If you want to communicate with Thomas, please type in French.

I think you can answer JamOrHam's question below if you don't mind please.

**Reply by Thomasr205** [reply]
@psonnera Hello.  I confirm that, even with data, I have the same problem.

Bonjour. Je confirme que, même avec des données, j'ai le même problème.

**Reply by Thomasr205** [reply]
@psonnera : "Is there a tag to add in debug to see which notifications are not seen as legit by xDrip+?"
Is it a question for me ?

**Reply by tloczekt** [reply]
Maybe the problem with Xdrip catching notifications is that we are using local (not English version of Minimed Mobile so notifications are  generated also in local language). For my case is polish language,  Thomasr205 uses french as I see on the screnshoot. What do you think?

**Reply by Cheun1** [reply]
@tloczekt @Thomasr205  j'ai une idée .....1/ajoute un clavier anglais dans ton tél 2/ ensuite dans android applications trouve Minimed Mobile et là tu dois pouvoir changer la langue de l'app. Change la et voyons .

**Reply by tloczekt** [reply]
@Cheun1 , @Thomasr205 : Cheun1 was totally right!!!  When I set main language for phone (I couldn't find option for single app - MinimeMed) it is working. 
This is not solution for my kid who doesn't know English yet (6years old), but this is definitely right way to find the root of problem.
Notifications are recognized only in English language.

**Reply by tloczekt** [reply]
@Cheun1 thank you so much for your suggestion. At meantime I will try to find way set language only for Minimed Mobile app exclude whole Android system language.

**Reply by Navid200** [reply]
This is perfect troubleshooting.  Thanks for identifying this.

Would you please have a look at the following to see if you can only change the language of the app that xDrip is reading from to English while leaving the main phone language to whatever you need?
https://support.google.com/android/answer/12395118?hl=en

**Reply by Cheun1** [reply]
@Navid200 we will try an another way , when we change the phone language in english all is ok , but it is not a good solution , we will try this --->languages ​​and input / manage keyboards / settings / languages ​​/ add (english) and I think that then in Minimedl Mobile we will have the choice of language ....I wait the test of @Thomasr205

**Reply by jamorham** [reply]
I did put support in I thought for internationalization with companion mode but I will have to investigate why that isn't working...

**Reply by jamorham** [reply]
Could we get a screenshot of what the notification looks like in non-english when it contains glucose data? thanks

**Reply by tloczekt** [reply]
@jamorham BG numbers are universal, but active insuline title and units are in local language. 
I have Android 12 and language settings are not so extended like Android 13. I have no such menu feature (like link  @Navid200 mentioned) in Android 12. Do you know can I upgrade already to Android 13  for Minimed Mobile? I heard about incompatibility on early stage.

**Reply by Cheun1** [reply]
@jamorham French 
![image](https://user-images.githubusercontent.com/41595891/212962303-efbebd9e-35a6-42fc-b004-06e2acb123b6.png)

**Reply by tloczekt** [reply]
@jamorham  maybe the arrow trend is problematic? We use 780g pump.  I see that @Cheun1 has no such feature.

**Reply by jamorham** [reply]
Please try the Jan 17th Nightly and let me know if this works better: https://github.com/NightscoutFoundation/xDrip/releases/download/2023.01.17/xDrip-plus-20230117-3d543b5.apk

**Reply by Navid200** [reply]
@tloczekt @Cheun1 Would you please try the new release?   Please see above.

**Reply by Cheun1** [reply]
yeah @jamorham all is ok with the last update .

**Reply by Thomasr205** [reply]
Tout fonctionne. Vous êtes des pros. Bravo les développeurs !!!

**Reply by Navid200** [reply]
Thomas's post translated by Google:

Everything is working. You are pros. Well done developers

**Reply by tloczekt** [reply]
@Navid200 , @jamorham  good news. It is working for me too. Great job developers. Very fast reaction. I'm impressed . Well done.

**Reply by jamorham** [reply]
Great to hear that the problem is resolved. Thanks for the testing.

**Comment by jamorham** [comment]
Do we have a play store link for the app?

**Reply by Cheun1** [reply]
https://play.google.com/store/apps/details?id=com.medtronic.diabetes.minimedmobile.eu&fbclid=IwAR1sP9bkN8M8sMCxzhnk1V2QAy8I-Gu5LzZ-SYT-9sS_dwaovtNjUxDiJKM

**Comment by tloczekt** [comment]
Polish language

wt., 17 sty 2023, 17:38 użytkownik JamOrHam ***@***.***>
napisał:

**Reply by tloczekt** [reply]
![tempFileForShare_20230117-174147_685232325962189](https://user-images.githubusercontent.com/71193996/212961094-90afc42e-35d3-4f79-85d9-2922262b89fd.jpg)
Polish language

**Comment by Thomasr205** [comment]
Un petit retour : En mettant le téléphone en anglais, ça fonctionne. 
Impossible de choisir l'anglais uniquement pour minimed mobile. 
Ensuite, j'ai changé le clavier en anglais et remis mon tel en français et ça ne fonctionne plus de nouveau.

**Reply by BUGWIZZ** [reply]
I tried to switch my phone to english but i Still get errors in the log.
![SmartSelect_20230427_182055_xDrip+](https://user-images.githubusercontent.com/121064009/234969304-9994bfa3-2786-420d-8287-d8a3574389d4.jpg)
I m unable to remove the double authentication from my minimed connect accourt dedicated to xdrip.( fr based )
How can i fix that?

**Reply by tloczekt** [reply]
The easiest way I see is to create Carelink  partner therapy new account  related to the your patient  account.  New partner account should be without dual authentication. Then you can use new Carelink login and pass on Xdrip as well

**Reply by Navid200** [reply]
@BUGWIZZ Why have you posted in this thread and have not opened a new thread?
Your screenshot shows CareLink.

**Reply by BUGWIZZ** [reply]
I have trouble connecting a pump 780g (closed loop) with xdrip. So the only discussion i fond was this one. Fr accounts are the issue?

**Reply by Navid200** [reply]
@BUGWIZZ This discussion did it's job.  It was about the Companion app mode.  It raised the issue that in some areas where the until is written as mg/dl instead of mg/dL, Companion app mode did not work.

It was addressed by the xDrip release on January 17, 2023.

You are raising a concern with CareLink follow, a different part of xDrip. It deserves it's own thread.  Anyone can open a new thread.  But, you have chosen to use an existing thread that is considered to have been addressed already.  As a result, it is not going to get the attention that it deserves.

Your choice.

**Reply by BUGWIZZ** [reply]
Thank you . as the carelink follower fails probablely due to unremovable dual authentication for french accounts i switched to"companion app". Thaïs works fine for my 780g pump. I was looking for a solution since i stoped using libre+miaomiao2 2 years ago.
